moto/backend_index.py

```diff
@@ -45,6 +45,7 @@
     ("directconnect", re.compile("https?://directconnect\\.(.+)\\.amazonaws\\.com")),
     ("dms", re.compile("https?://dms\\.(.+)\\.amazonaws\\.com")),
     ("ds", re.compile("https?://ds\\.(.+)\\.amazonaws\\.com")),
+    ("dsql", re.compile("https?://dsql\\.(.+)\\.api\\.aws")),
```

Comment:
DSQL Endpoint is sub domain of api.aws like `dsql.us-east-1.api.aws` not amazonaws.com

Reply:
Interesting. I wonder whether that's an outlier, or whether they're going to host other (existing?) services there as well.